analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -126,9 +126,12 @@ class Gradle(
     override fun resolveDependencies(definitionFile: File): ProjectAnalyzerResult? {
         val gradleSystemProperties = mutableListOf<Pair<String, String>>()
 
-        // Usually, the Gradle wrapper handles applying system properties from Gradle properties. But as we directly use
-        // the tooling API, we need to manually load Gradle properties and apply any system properties. Limit the lookup
-        // to the current user's Gradle properties file for now.
+        // Usually, the Gradle wrapper's Java code handles applying of system properties defined in a Gradle properties
```

Comment:
"handles applying of": should it be either "handles the application of" or "handles appyling"?

Reply:
Done.